src/zmalloc.c

```diff
@@ -674,7 +731,8 @@ size_t zmalloc_get_frag_smallbins(void) {
 }
 
 int zmalloc_get_allocator_info(size_t *allocated, size_t *active, size_t *resident,
-                               size_t *retained, size_t *muzzy, size_t *frag_smallbins_bytes)
+                               size_t *retained, size_t *muzzy, size_t *frag_smallbins_bytes,
+                               unsigned int lua_arena)
```

Comment:
let's document the arguments to this function.
specifically the use of `lua_arena`, or maybe it should be renamed to `exclude_arena`.